### DISCUSSION TITLE
Hi team!

### DISCUSSION THREAD

**Discussion by MMikhailova** [lab-brussels-1/HYF-Redesign-Group-3#30] [Ideas]
I was thinking of our design and I have a suggestion.

Since we were advised to use semantic elements (header, navigation, section, article, footer), may be we can use them to divide our design?

![image](https://user-images.githubusercontent.com/98428800/163174694-dbd8a525-9c66-4868-b81e-52c61a52fddd.jpeg)

So that we could practice semantics and it helps with task division.

Please let's me know what do you think here or in the meeting.🤗

**Comment by FennyWilriani** [comment]
Yes sure, we can choose the part and identify it with semantics 🤓
